### ISSUE TITLE
getting similar  error https://veracrypt.codeplex.com/discussions/661382

### ISSUE THREAD

**Issue by cerberii** [veracrypt/VeraCrypt#245]
VeraCrypt Volume Creation Wizard

The system cannot find the path specified

Source: VeraCrypt::Elevator::BackupEfiSystemLoader::336

I dont know exactly what is the cause,  I have encrypted the drive without any problems just a few hours ago, I did a wipe and created an lvm linux group of paritions using cryptsetup and now it throws this error when trying to encrypt the same windows install again, this system is multibooting.  Seems I have to first encrypt with veracrypt otherwise Veracrypt is throwing this error as it has some kind of trouble accessing the EFI?  I will wipe again and first encrypt veracrypt volume, then install linux fde parts

I can access efi in windows and tested writing to it in an elevated command prompt.

**Comment by cerberii** [comment]
cause: unknown

remedy, reinstall and start from scratch

**Comment by chaos-prevails** [comment]
I have the same issue when trying to system encrypt. Unfortunately reinstalling VC didn't solve it.

I use: Veracrypt 1.22, and Windows 7 Professional 64b with UEFI-CSM boot

Looking at the code the function BackupEfiSystemLoader seems to need access to EFI\Boot\

Prior to trying to encrypt, I changed from MBR/BIOS to GPT/UEFI (also to speed up Veracrypt boot time), following this guide: http://www.hasper.info/repair-a-destroyed-windows-7-uefi-boot-sector/

Doing so, didn't create EFI\Boot, neither the file EFI\Boot\bootx64.efi

To solve this issue (in my case): 
- create \EFI\Boot
- copy \EFI\Microsoft\Boot\bootmgfw.efi to \EFI\Boot\bootx64.efi
- restart the system encryption process.

I successfully encrypted my system, and so far, has survived 2 restarts :)
